src/main/java/sirius/db/mixing/properties/StringListProperty.java

```diff
@@ -113,12 +122,18 @@ protected Object transformFromMongo(Value object) {
 
     @Override
     protected Object transformFromJDBC(Value object) {
-        return Arrays.asList(object.coerce(String[].class, EMPTY_STRING_ARRAY));
+        if (oma.getDatabase(descriptor.getRealm()).hasCapability(Capability.LISTS)) {
```

Comment:
could we please cache this flag?

Reply:
maybe even call this in the constructor? or a verify / link run

Reply:
maybe also extend the description of the class javadoc of this class an StringList to state that it can be used with JDBC databases, how it is handled and what limits are imposed.

Reply:
could not be executed in constructor or link, because OMA was not ready then